src/main/java/org/broadinstitute/hellbender/tools/copynumber/formats/collections/AbstractRecordCollection.java

```diff
@@ -184,7 +184,7 @@ private SAMFileHeader getHeader() throws FileNotFoundException {
 
         @Override
         protected boolean isCommentLine(final String[] line) {
-            return line.length > 0 && line[0].startsWith(COMMENT_PREFIX);
+            return line.length > 0 && (line[0].startsWith(COMMENT_PREFIX) || line[0].startsWith("#"));
```

Comment:
Also going to veto this.  The `AbstractRecordCollection` should be a very strictly defined format.  If you have external GT files that we need to remove comment lines from a single time, let's do that externally rather than relax the format.

Reply:
Removed it.  I'll modify the validation pipeline.  I worry that this will be a problem later on, when we want to support comments and sequence dictionaries.

Done.